### ISSUE TITLE
Browser Support

### ISSUE THREAD

**Issue by jvansant** [canadaduane/vscode-notes#18]
It would be amazing to port this extension into the [browser app](https://vscode.dev).

VSCode supplies [this page](https://code.visualstudio.com/api/extension-guides/web-extensions) regarding web extensions. I'll be looking into it when I've got a chance.

**Comment by canadaduane** [comment]
Yeah, that would be cool. I don't have time to port right now, but would be happy to support with any questions.
